lib/java/build.gradle

```diff
@@ -60,7 +60,6 @@ if (Boolean.parseBoolean(project.release)) {
 // Keeping the rest of the build logic in functional named scripts for clarity
 apply from: 'gradle/environment.gradle'
 apply from: 'gradle/sourceConfiguration.gradle'
-apply from: 'gradle/additionalArtifacts.gradle'
```

Comment:
this file is removed and not needed and superseded by https://github.com/apache/thrift/pull/2601/files#diff-da5bde3bdf21a82cc6725ecd18f6f9f7d7c1dbcf7789e264892b51053a330a36R41-R43